tools/topology/topology1/sof/pipe-dts-codec-eq-iir-playback.m4

```diff
@@ -0,0 +1,113 @@
+# Low Latency Passthrough with DTS codec Pipeline and PCM
+#
+# Pipeline Endpoints for connection are :-
+#
+# host PCM_P --> B0 --> DTS Codec --> B1 --> EQ 0 --> B2 --> sink DAI0
+
+# Include topology builder
+include(`utils.m4')
+include(`buffer.m4')
+include(`pcm.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`codec_adapter.m4')
+include(`bytecontrol.m4')
+include(`eq_iir.m4')
+
+#
+# Controls
+#
+
+#
+# DTS Codec
+#
+
+include(`dts_codec_adapter.m4')
+
+#
+# IIR EQ
+#
+define(DEF_EQIIR_COEF, concat(`eqiir_coef_', PIPELINE_ID))
+define(DEF_EQIIR_PRIV, concat(`eqiir_priv_', PIPELINE_ID))
+
+# define filter. eq_iir_coef_flat.m4 is set by default
+ifdef(`PIPELINE_FILTER1', , `define(PIPELINE_FILTER1, eq_iir_coef_flat.m4)')
+include(PIPELINE_FILTER1)
+
+# EQ Bytes control with max value of 255
+C_CONTROLBYTES(DEF_EQIIR_COEF, PIPELINE_ID,
+    CONTROLBYTES_OPS(bytes, 258 binds the mixer control to bytes get/put handlers, 258, 258),
+    CONTROLBYTES_EXTOPS(258 binds the mixer control to bytes get/put handlers, 258, 258),
+    , , ,
+    CONTROLBYTES_MAX(, 1024),
+    ,
+    DEF_EQIIR_PRIV)
+
+#
+# Components and Buffers
+#
+
+ifdef(`CA_SCHEDULE_CORE',`', `define(`CA_SCHEDULE_CORE', `SCHEDULE_CORE')')
+
+# Host "Playback with codec adapter" PCM
+# with DAI_PERIODS sink and 0 source periods
+W_PCM_PLAYBACK(PCM_ID, Passthrough Playback, DAI_PERIODS, 0, SCHEDULE_CORE)
+
+W_CODEC_ADAPTER(0, PIPELINE_FORMAT, DAI_PERIODS, DAI_PERIODS, CA_SCHEDULE_CORE,
+        LIST(`          ', "CA_SETUP_CONTROLBYTES_NAME_PIPE", "CA_RUNTIME_CONTROLBYTES_NAME_PIPE"))
```

Comment:
there is no CA_RUNTIME_CONTROLBYTES_NAME_PIPE now

Reply:
Yes, I've already removed it now.